backend/question/routes/questionRoutes.ts

```diff
@@ -0,0 +1,137 @@
+import express from 'express';
+import { check, validationResult, body } from 'express-validator';
+import Question from "../models/Question";
+
+/**
+ * Router for the question service.
+ */
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+    res.send('Hello from question service!');
+});
+
+// Create a new question
+router.post('/create', [
+    check('title').isString().isLength({ min: 1 }),
```

Comment:
Just for ease of reading, maybe we can move the validation rules to a variable or another file?